api/src/main/java/org/openmrs/api/db/DelegatingFullTextSession.java

```diff
@@ -0,0 +1,138 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * 
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.api.db;
+
+import java.io.Serializable;
+
+import org.apache.commons.logging.Log;
```

Comment:
We do not recommend logging with apache commons: https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-loggingLogging